chirp/wxui/main.py

```diff
@@ -1835,7 +1836,12 @@ def _do_network_query(self, query_cls):
             self.add_editorset(editorset)
 
     def _menu_query_rr(self, event):
-        self._do_network_query(query_sources.RRQueryDialog)
+        if radioreference.HAVE_SUDS:
```

Comment:
So, this (optionality and flag) is legacy and not actually because of linux, but macOS. Long ago, the frozen-in-time macos build runtime was missing the suds library and there wasn't an easy way to fix it. As of the new UI/python3 rewrite, that's no longer the case. Because of this optionality, it was also (mistakenly) left out of the build requirements, but not intentionally.

https://github.com/kk7ds/chirp/blob/master/setup.py#L15

Point is, it is now required, we should assume it's present. I'd rather we just remove all this optional import stuff and treat it like anything else.